game/pokemon.py

```diff
@@ -17,16 +19,21 @@ def __init__(
         name: str,
         dex_number: int,
         types: List[Element],
-        total_health: Optional[int] = None
+        total_health: Optional[int] = None,
     ):
+        """ads"""
```

Comment:
This doesn't tell me anything

Reply:
Changed to Property of Pokemon initialized